lib/util.ml

```diff
@@ -3010,8 +3154,9 @@ let short_f_month m =
 
 type auth_user = { au_user : string; au_passwd : string; au_info : string }
 
+(* TODO provide bname *)
```

Comment:
Need to be done before merging.

Reply:
Is it bname which is needed there or conf.paths.password_files_dir? (or equivalent)
In both cases, we need to add conf as a parameter to the chain of calling points! correct?

Reply:
Do not know yet, it was just a memo for me so I could replicate old behavior.